site/content/docs/5.2/components/footer.md

```diff
@@ -105,7 +105,7 @@ The footer is a container with a `.footer-terms` class. This footer bar is **man
 
 ## Behavior
 
-Footer are designed to be responsive on all devices.
+The footer component is designed to be responsive on all devices.
```

Comment:
In Orange navbar: "Orange navbars are designed to be responsive on all devices." (plural form)